costs.css

```diff
@@ -144,117 +144,123 @@ input {
 }
 
 @media (max-width: 850px) {
-.div-value input {
-  margin-bottom: 20px;
-}
-
-.newinptxt {
-  width: 12vh;
-}
-
-.newinpval {
-  width: 80%;
-}
-
-.newinpdata {
-  width: 80%;
-}
-
-.text-container p, .dataCont-container p, .cost-container p {
-  font-size: 24px;
-}
-
-.img-container {
-  display: flex;
-  justify-content: center;
-  margin: 0;
-}
-
-.text-container {
-  width: 40%;
-}
-
-.text-container {
-  word-wrap: break-word;
-}
-
-.div-value {
-  display: flex;
-  justify-content: center;
-  flex-direction: column;
-}
-
-.but1 {
-  width: 40%; 
-}
-
-.mark-container {
-  display: flex;
-  align-items: center;    
-  justify-content: space-between;
-  margin-bottom: 10px;
-} 
-
-#content-page {        
-  height: 300px;
-  background: rgba(0, 0, 0, 0.3);
-} 
+  .div-value input {
+    margin-bottom: 20px;
+  }
+
+  .newinptxt {
+    width: 12vh;
+  }
+
+  .newinpval {
+    width: 80%;
+  }
+
+  .newinpdata {
+    width: 80%;
+  }
+
+  .text-container p, 
+  .dataCont-container p, 
+  .cost-container p {
+    font-size: 24px;
+  }
+
+  .img-container {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+
+  .text-container {
+    width: 40%;
+  }
+
+  .text-container {
+    word-wrap: break-word;
+  }
+
+  .div-value {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+  .but1 {
+    width: 40%; 
+  }
+
+  .mark-container {
+    display: flex;
+    align-items: center;    
+    justify-content: space-between;
+    margin-bottom: 10px;
+  } 
+
+  #content-page {        
+    height: 300px;
+    background: rgba(0, 0, 0, 0.3);
+  } 
 } 
 
 @media (max-width: 610px) {
-.main-div h1 {
-  font-size: x-large;  
-  font-style: oblique;
-}
-
-.text-container p, .dataCont-container p, .cost-container p {
-  font-size: 33px;
-  display: flex;
-  justify-content: center;
-}
-
-.text-container, .dataCont-container, .cost-container {
-  display: flex;
-  width: 100%;
-  justify-content: center;
-}
-
-.dateP-location {
-  margin: 4vh 0;
-}
-
-.img-container {
-  display: flex;
-  width: 70%;
-  justify-content: space-between;
-}
-
-.cash-location {
-  margin: 2vh 0;
-}
-
-.but {
-  margin: 2vh 0;
-  width: 70px;
-  height: 70px;
-}
-
-.but1 {
-  width: 80%;
-}  
-
-#content-page {     
-  height: 500px;
-  overflow-y: auto;
-  display: flex;
-  align-items: center;
-  font-size: large;
-  flex-direction: column;
-  background: rgba(0, 0, 0, 0.3);
-}
-
-.mark-container {
-  flex-direction: column; 
-}   
+  .main-div h1 {
+    font-size: x-large;  
+    font-style: oblique;
+  }
+
+  .text-container p, 
+  .dataCont-container p, 
+  .cost-container p {
+    font-size: 33px;
+    display: flex;
+    justify-content: center;
+  }
+
+  .text-container, 
```

Comment:
Why you don't use one class?

Reply:
because if I use class 1, the layout will be displayed incorrectly